browse-scaler/localstack/docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: "2"
+services:
+  localstack:
+    image: localstack/localstack:latest
+    environment:
+      DEBUG: 1
+      SERVICES: "lambda,elasticache,apigateway"
+      LAMBDA_EXECUTOR: docker
+      DOCKER_HOST: unix:///var/run/docker.sock
+      # DATA_DIR: /tmp/localstack/data
```

Comment:
Remove commented out code or document when we'd want to uncomment this.

Reply:
removing the localstack files for now - I got carried away and made this too big of a PR